jekyll/index.html

```diff
@@ -9,22 +9,55 @@
       <div class="pure-u-1 pure-u-md-3-5 main-block">
         <h1>Efficient and expressive programming.</h1>
         <h2>
-            Nim is a systems and applications programming language.
-            Statically typed and compiled, it provides unparalleled performance
-            in an elegant package.
+          Nim is a statically typed, compiled, garbage-collected systems programming language
+          with a design that focuses on efficiency, expressiveness,
+          and elegance, providing unparalleled
```

Comment:
Remove "providing unparalleled performance in an elegant package", it's braggy and doesn't add anything to what was said earlier.